pkg/services/sqlstore/temp_user.go

```diff
@@ -132,3 +133,15 @@ func GetTempUserByCode(query *models.GetTempUserByCodeQuery) error {
 	query.Result = &tempUser
 	return err
 }
+
+func ExpireOldUserInvites(cmd *models.ExpireTempUsersCommand) error {
+	return inTransaction(func(sess *DBSession) error {
+		var rawSql = "UPDATE temp_user SET status = ?, updated = ? WHERE created <= ? AND status = ?"
+		if result, err := sess.Exec(rawSql, string(models.TmpUserExpired), time.Now().Unix(), cmd.OlderThan.Unix(), string(models.TmpUserInvitePending)); err != nil {
+			return err
+		} else if cmd.NumExpired, err = result.RowsAffected(); err != nil {
+			return err
+		}
```

Comment:
The `else` is non-idiomatic I believe, with Go error handling.
```suggestion
		}
		if cmd.NumExpired, err := result.RowsAffected(); err != nil {
			return err
		}
```

Reply:
Omitting the `else` actually results in both `err` and `result` as unresolved references 🕵️  Since they no longer belong to the same block once splitting

Reply:
Ah, I didn't take into account that `result` is defined on line 140 and used on line 142. With my change though, `err` should be defined at least.